### DISCUSSION TITLE
Can I push from local to local?

### DISCUSSION THREAD

**Discussion by martin-josef** [welaika/wordmove#633] [Q&A]
Hello everyone, for some testing I'd like to push from one local db to another local one. Is there a way to do it / to skip ssh? Kind regards, Martin

**Comment by alessandro-fazzi** [comment]
If your intent is to test wordmove self, then my answer simply is: no, it's not possible.

But if your intent is to test something else using wordmove, then I'll add that it shouldn't be worth to pickup wordmove for a simple local clone, since it could be a 2-line shell script ;)
